File: app/src/main/java/com/moneycollect/example_java/utils/CardImageUtils.java
package com.moneycollect.example_java.utils;

import android.annotation.SuppressLint;
import android.content.Context;
import android.widget.ImageView;

import com.moneycollect.example_java.R;

public class CardImageUtils {
    @SuppressLint("UseCompatLoadingForDrawables")
    public static void setCardImg(Context context, ImageView iv, String cardBrand) {
        if (cardBrand.contains("Visa")){
            iv.setBackground(context.getResources().getDrawable(R.drawable.mc_card_visa));
        }else  if (cardBrand.contains("Master")){
            iv.setBackground(context.getResources().getDrawable(R.drawable.mc_card_mastercard));
        }else if (cardBrand.contains("Ame")){
            iv.setBackground(context.getResources().getDrawable(R.drawable.mc_card_ae));
        }else if (cardBrand.contains("JCB")){
            iv.setBackground(context.getResources().getDrawable(R.drawable.mc_card_jcb));
        }else if (cardBrand.contains("Dinne")){
            iv.setBackground(context.getResources().getDrawable(R.drawable.mc_card_dinner));
        }else if (cardBrand.contains("Discover")){
            iv.setBackground(context.getResources().getDrawable(R.drawable.mc_card_discover));
        }else if (cardBrand.contains("Maestro")){
            iv.setBackground(context.getResources().getDrawable(R.drawable.mc_card_maestro));
        }
    }
}

File: app/src/main/java/com/moneycollect/example/utils/CardImageUtils.kt
package com.moneycollect.example.utils

import android.annotation.SuppressLint
import android.content.Context
import android.widget.ImageView
import com.moneycollect.android.R


@SuppressLint("UseCompatLoadingForDrawables")
fun setCardImg(context: Context?, iv: ImageView?, cardBrand: String?) {
    when {
        cardBrand!!.contains("Visa") -> {
            iv!!.background = context?.resources?.getDrawable(R.drawable.mc_card_visa)
        }
        cardBrand.contains("Master") -> {
            iv!!.background = context?.resources?.getDrawable(R.drawable.mc_card_mastercard)
        }
        cardBrand.contains("Ame") -> {
            iv!!.background = context?.resources?.getDrawable(R.drawable.mc_card_ae)
        }
        cardBrand.contains("JCB") -> {
            iv!!.background = context?.resources?.getDrawable(R.drawable.mc_card_jcb)
        }
        cardBrand.contains("Dinne") -> {
            iv!!.background = context?.resources?.getDrawable(R.drawable.mc_card_dinner)
        }
        cardBrand.contains("Discover") -> {
            iv!!.background = context?.resources?.getDrawable(R.drawable.mc_card_discover)
        }
        cardBrand.contains("Maestro") -> {
            iv!!.background = context?.resources?.getDrawable(R.drawable.mc_card_maestro)
        }
    }
}